CHANGELOG.md

```diff
@@ -1,3 +1,123 @@
+## [Talos 1.8.2](https://github.com/siderolabs/talos/releases/tag/v1.8.2) (2024-10-28)
+
+Welcome to the v1.8.2 release of Talos!
+
+Starting with Talos v1.8.0, only standard assets would be published as github release assets. These include:
```

Comment:
i thought we removed this

Reply:
no, this was for 1.9. I think it's fine for 1.8